sql.txt

```diff
@@ -1,15 +1,24 @@
 (1) Using the Customer Table, select the CustomerName, ContactName, and Country.  Use ORDER BY to order by Country.  
 Use LIMIT and OFFSET to get entries 11 through 20. Paste your SQL statement below. 
+select CustomerName, ContactName, Country FROM Customers order by country limit 10 offset 10;
 
 (2) Select all columns from the Customer table where the ContactName starts with A.  Paste your SQL statement below.
+select * from Customers where ContactName like "A%";
 
 (3) Select all columns from the OrderDetails table where the ProductID is 51 and the quantity is greater than 10.
 Paste your SQL statement below.
+select * from OrderDetails where ProductID = 51 and quantity > 10;
 
 (4) Insert 3 rows into the Products table.  Note that you will have to specify a valid SupplierID and CategoryID, 
 corresponding to rows from the Supplier and Category tables.  Paste your three SQL statements below.
+update Products values 
+Insert into Products (ProductID, ProductName, SupplierID, CategoryID, Unit, Price) values (78, 'a', 100, 100, "100boxes", 100);
+Insert into Products (ProductID, ProductName, SupplierID, CategoryID, Unit, Price) values (78, 'b', 100, 100, "1000boxes", 100);
+Insert into Products (ProductID, ProductName, SupplierID, CategoryID, Unit, Price) values (78, 'c', 100, 100, "10000boxes", 100);
```

Comment:
This way we don't need to repeat INSERT INTO....
Insert into Products (ProductID, ProductName, SupplierID, CategoryID, Unit, Price) values 
    (78, 'a', 100, 100, "100boxes", 100),
    (78, 'b', 100, 100, "1000boxes", 100),
    (78, 'c', 100, 100, "10000boxes", 100);

Reply:
That is what I initially tried to do but somehow it won't let me. I still get an error message saying "Error 1: could not execute statement due to a constraint failure (19 UNIQUE constraint failed: Products.ProductID)" Could you tell me what the issue is?

Reply:
Mmmm, this is very weird. I'm trying by myself and it seems it doesn't work in w3school, however it works when I use SQL in my terminal. It seems like a bug of w3school.